packages/presentation/lib/screen/home/home_screen.dart

```diff
@@ -0,0 +1,150 @@
+import 'package:flutter/material.dart';
```

Comment:
pls, do some separate files

Reply:
I doing it now in next task.

Reply:
@vladimir-hodaryonok-idf for home_screen now it's ok. but if we need to separate the splash screen into widget it should be done in this task